src/pages/MarketPage/components/AllItemsSection.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
+import { useProducts } from "../../../hooks/useProducts";
+import { useAllPageSize } from "../../../hooks/useAllPageSize";
+import ItemCard from "./ItemCard";
+import sortIconMobile from "../../../assets/icons/ic_sort_mobile.svg";
+import sortIconArrowDown from "../../../assets/icons/ic_arrow_down.svg";
+import searchIcon from "../../../assets/icons/ic_search.svg";
+import DropdownList from "../../../components/UI/DropdownList";
+import PaginationBar from "../../../components/UI/PaginationBar";
+
+function AllItemsSection() {
+  const [orderBy, setOrderBy] = useState("recent");
+  const [page, setPage] = useState(1);
+  const [keyword, setKeyword] = useState("");
+  const [isDropdownVisible, setIsDropdownVisible] = useState(false);
+  const pageSize = useAllPageSize();
+
+  const { itemList, totalCount, loading, error } = useProducts({
+    orderBy,
+    page,
+    pageSize,
+    keyword,
+  });
+
+  const totalPageNum = Math.ceil(totalCount / pageSize);
+
+  const handleSortSelection = (sortOption) => {
+    setOrderBy(sortOption);
+    setIsDropdownVisible(false);
+    setPage(1);
+  };
+
+  const handleInputChange = (event) => {
+    setKeyword(event.target.value);
+  };
+
+  const handleSearch = () => {
+    setPage(1);
+  };
+
+  const handleKeyPress = (event) => {
+    if (event.key === "Enter") {
+      handleSearch();
+    }
+  };
+
+  const convertToKorean = (orderBy) => {
+    switch (orderBy) {
+      case "recent":
+        return "최신 순";
+      case "favorite":
+        return "좋아요 순";
+      default:
+        return "최신 순";
+    }
+  };
+
+  const toggleDropdown = () => {
+    setIsDropdownVisible(!isDropdownVisible);
+  };
+
+  const onPageChange = (pageNumber) => {
+    setPage(pageNumber);
+  };
```

Comment:
```suggestion
  const handlePageChange = (pageNumber) => {
    setPage(pageNumber);
  };
```

on prefix는 보통 props로 받는 핸들러같은것에 많이 붙히는 prefix입니다.